mobile/evidence/ios/Features/Tests/LoginTests/InputValidatorTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  inputValidatorTests.swift
+//  
+//
+//  Created by Cris Messias on 16/05/24.
+//
+
+import XCTest
+@testable import Login
+@testable import Models
+import Dependencies
+
+final class InputValidatorTests: XCTestCase {
```

Comment:
Maybe you could add more tests for edge cases such as `@example.com`, `test@`, `a@b`, `a@b.com.br.co.de`

Reply:
Nice idea. The test failed in this last example...